apps/st2-history/template.html

```diff
@@ -113,7 +113,14 @@ <h2 class="st2-flex-table__caption-title">{{ group.period | date:'fullDate':'UTC
                   {{ record.trigger.type }}
                 </span>
               </span>
-              <span ng-switch-default>Manual</span>
+              <span ng-switch-default>
+                <span class="st2-history__column-app-name">
+                  Manual
```

Comment:
Btw, why do we say "manual" and not just display the user name? What other possibles values are there besides manual?